### DISCUSSION TITLE
Password protected warps

### DISCUSSION THREAD

**Discussion by evanonym0us** [Zrips/CMI#7553] [Enhancement]
Allow password protecting a warp, OR allow disabling of the warp module in modules.yml so we can use a different plugin that has this feature.

**Comment by T14D3** [comment]
Both things are already possible.
For password protected warps, create a custom alias that checks for the password, and use asConsole! to warp the player to the warp and NOT give the player permission for the warp
For disabling the warp function, alias.yml

**Reply by ETechTechnologies** [reply]
You could make it easier to set a password, like in the warp menu

**Reply by MrCrazys** [reply]
> Both things are already possible. For password protected warps, create a custom alias that checks for the password, and use asConsole! to warp the player to the warp and NOT give the player permission for the warp For disabling the warp function, alias.yml

This doesn't work if you want it to apply in the warp menu. Sure, it may work for some cases, but it's not really the best solution. I think it's a fair request.
